### DISCUSSION TITLE
Add container-queries in CDN

### DISCUSSION THREAD

**Discussion by Timonney** [tailwindlabs/tailwindcss#10598] [Ideas]
Hi,
I  just tried to add the container-queries in the CDN with the follwing link :
https://cdn.tailwindcss.com/?plugins=container-queries

But it returns :

```
console.error("Unknown plugin: `container-queries`
Available plugins: `aspect-ratio`, `forms`, `line-clamp`, `typography`")
```
Is it possible to add this plugin to the CDN ?

**Comment by mnik01** [comment]
Hello 👋 
Was solved at #12667
